static/app/views/dashboardsV2/utils.tsx

```diff
@@ -32,3 +37,26 @@ export function eventViewFromWidget(
     environment: environments,
   });
 }
+
+type FeatureProps = {
+  organization: Organization;
+  children: React.ReactNode;
+};
+
+export const DashboardBasicFeature = ({organization, children}: FeatureProps) => {
```

Comment:
should we move this component to a separate file? I believe we usually only write functions in `utils` files